pkg/tuple/tuple.go

```diff
@@ -63,6 +63,21 @@ func GetObjectRelationAsString(objectRelation *openfgapb.ObjectRelation) string
 	return objectRelation.GetObject()
 }
 
+// GetRelationReferenceAsString returns team#member, or team:*, or an empty string if the input is nil.
+func GetRelationReferenceAsString(rr *openfgapb.RelationReference) string {
```

Comment:
We have methods in the `typesystem` package to deal with `RelationReferences`. The RelationReference structure was introduced as a result of adding type restrictions and is technically part of the typesystem ecosystem. For that reason, can we place this in the `typesystem` package instead?

Then the tests below can change to:

```
func TestGetRelationReferenceAsString(t *testing.T) {
	require.Equal(t, "", GetRelationReferenceAsString(nil))
	require.Equal(t, "team#member", GetRelationReferenceAsString(DirectRelationReference("team", "member")))
	require.Equal(t, "team:*", GetRelationReferenceAsString(WildcardRelationReference("team")))
}
```

We should reserve the `tuple` package for operations/helpers with relationship tuples structures, not typesystem related stuff.